src/content/web/libraries.md

```diff
@@ -4,46 +4,41 @@ short-title: Web libraries
 description: Libraries and packages that can help you write Dart web apps.
 ---
 
-The [Dart SDK][] contains [dart:html][] and other libraries
-that provide low-level web APIs.
-You can supplement or replace these APIs using web packages.
+Dart provides several packages and libraries to support
+web app development, the recommended option being [package:web][].
+The [Dart SDK][] also contains other libraries that provide low-level web APIs.
 
-[Dart SDK]: /tools/sdk
-[dart:html]: {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-html/dart-html-library.html
+## Web solutions
 
+[Migrate to package:web][migrate]
+: Learn how to migrate to `package:web`
+  from Dart's previous web solution, [dart:html][].
 
-## SDK libraries
+[package:web API reference][package:web]
+: Complete reference documentation for `package:web` on pub.dev.
 
-The Dart SDK contains dart:html and other libraries
-that provide low-level web APIs.
+[JavaScript interoperability documentation](/interop/js-interop)
+: Interact with existing JavaScript or TypeScript libraries
+  using Dart's JS interop support.
 
-[Build a web app with Dart](/web/get-started)
-: A quick overview of how to build, run, and debug a web app with Dart.
-
-[The dart:html documentation](/libraries/dart-html)
-: An example-driven tour of using the dart:html library.
-  Topics include manipulating the DOM programmatically,
-  making HTTP requests, and using WebSockets.
-
-[dart:html API reference][dart:html]
-: Complete reference documentation for the dart:html library.
+[The dart:js_interop documentation][js_interop]
+: Complete reference documentation for the `dart:js_interop` library.
 
+[Flutter web support][flutter-web]
+: The [Flutter framework][flutter] supports web development with Dart,
+  in addition to mobile, desktop, and embedded device support.
 
-## Web packages
-
-Many [packages](/guides/packages) support web development with Dart.
-In particular, the [Flutter framework][flutter] has [web support][flutter-web],
-in addition to mobile, desktop, and embedded device support.
+[Build a web app with Dart](/web/get-started)
+: A quick overview of how to build, run, and debug a web app with Dart.
 
-To find more libraries that support writing web apps, 
+To find more support for writing web apps, 
```

Comment:
"To find more support" reads to me as the link will lead to me docs not packages, so I think the previous text or some other modification is likely a bit more clear.

Perhaps:

```suggestion
To find other libraries that support the web platform,
```